sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Combine.java

```diff
@@ -435,6 +488,27 @@ public OutputT defaultValue() {
    */
   public abstract static class BinaryCombineFn<V> extends CombineFn<V, Holder<V>, V> {
 
+    /**
+     * Returns a {@code CombineFn} that uses the given {@code SerializableBiFunction} to combine
+     * values.
+     */
+    public static <V> BinaryCombineFn<V> of(SerializableBiFunction<V, V, V> combiner) {
+      return new BinaryCombineFn<V>() {
+        @Override
+        public V apply(V left, V right) {
+          return combiner.apply(left, right);
+        }
+      };
+    }
+
+    public static BinaryCombineFn<Long> foo() {
```

Comment:
Are these two functions (foo and bar) supposed to be here? Looks like testing code that was left behind.

Reply:
Oh, yes, I was validating what inference could and could not accept. It's now in the test file but I forgot to delete it here. Thanks.